src/alchemlyb/parsing/__init__.py

```diff
@@ -31,8 +31,13 @@ def wrapper(outfile, T, *args, **kwargs):
         dict_with_df = func(outfile, T, *args, **kwargs)
         for k in dict_with_df.keys():
             if dict_with_df[k] is not None:
+
+                if T < 273:
```

Comment:
Would it be better for this to be a warning?
I mean if T is a negative number then it is definitely an error but if it is just below 0 C then it is not strictly impossible.

Reply:
I can remove this since I took the liberty of adding it, but since you have a value of the gas constant that is hard coded to be in Kelvin and the docs are defined in Kelvin, this will always be true and catch the error that someone accidentally entered a value in C.